python/README.md

```diff
@@ -2,11 +2,17 @@
 
 Dev-nation (https://www.youtube.com/watch?v=QXeEoD0pB3E&list=PLsyeobzWxl7poL9JTVyndKe62ieoN-MZ3)
 
-THis above link gives a full course on Python. Navin Reddy is the teacher in the video. He simplifies every 
+This above link gives a full course on Python. Navin Reddy is the teacher in the video. He simplifies every 
 topic and teaches everything in such detail that no one else would explain it so really well.
 
 Dev-nation (https://www.w3schools.com/python/default.asp)
 
-Its one of the best websites I keep visiting to learn anything about coding. This website also gives us examples to try 
+It's one of the best websites I keep visiting to learn anything about coding. This website also gives us examples to try 
 them all by ourselves. We will not need to install any software to work on examples while studying on this website. 
-They have their own compilers. In that way its a very convenient website to learn any programming language.
\ No newline at end of file
+They have their own compilers. In that way its a very convenient website to learn any programming language.
```

Comment:
Add newer links

Reply:
Please leave out the introduction below the links " this guy ... " it's more opinion than anything related or which will help the app run i.e not official documentation